package.json

```diff
@@ -1,9 +1,11 @@
 {
   "scripts": {
-    "build": "spago build -u '--strict --stash +RTS -N2 -RTS'",
+    "build": "spago build -u '+RTS -N2 -RTS'",
     "test": "spago build && jest"
   },
   "devDependencies": {
+    "@babel/preset-env": "^7.18.10",
+    "babel-jest": "^28.1.3",
```

Comment:
Thanks, I wonder if use `NODE_OPTIONS=--experimental-vm-modules jest` like in https://github.com/nonbili/purescript-jest/pull/3/files, are these two deps still needed?

Reply:
I saw that, wasn't sure which you'd think was "cleaner". Tried applying this diff:

```diff
diff --git a/jest.config.js b/jest.config.js
index bdab438..2ade986 100644
--- a/jest.config.js
+++ b/jest.config.js
@@ -3,9 +3,6 @@
 
 module.exports = {
   testEnvironment: "jsdom",
-  transform: {
-      "^.+\\.(js|jsx)$": "babel-jest",
-  },
 
   // An array of directory names to be searched recursively up from the requiring module's location
   moduleDirectories: ["node_modules", "output"],
diff --git a/package.json b/package.json
index 95c0d36..64ce04e 100644
--- a/package.json
+++ b/package.json
@@ -1,11 +1,9 @@
 {
   "scripts": {
     "build": "spago build -u '+RTS -N2 -RTS'",
-    "test": "spago build && jest"
+    "test": "spago build && NODE_OPTIONS=--experimental-vm-modules jest"
   },
   "devDependencies": {
-    "@babel/preset-env": "^7.18.10",
-    "babel-jest": "^28.1.3",
     "jest": "^28.1.0",
     "jest-environment-jsdom": "^28.1.0"
   }
diff --git a/test/main.test.js b/test/main.test.js
index efd714a..04406bb 100644
--- a/test/main.test.js
+++ b/test/main.test.js
@@ -1,3 +1,3 @@
-const Test = require("Test.Main");
+import { main } from 'Test.Main';
 
-Test.main();
+main();
```

But looks like it doesn't work - maybe because this jest environment is set to `jsdom`?

```
> test
> spago build && NODE_OPTIONS=--experimental-vm-modules jest

[info] Build succeeded.
[warn] None of your project files import modules from some projects that are in the direct dependencies of your project.
These dependencies are unused. To fix this warning, remove the following packages from the list of dependencies in your config:
- psci-support
(node:2061657) ExperimentalWarning: VM Modules is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
 FAIL  test/main.test.js
  ● Test suite failed to run

    Jest encountered an unexpected token

    Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.

    Out of the box Jest supports Babel, which will be used to transform your files into valid JS based on your Babel configuration.

    By default "node_modules" folder is ignored by transformers.

    Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/ecmascript-modules for how to enable it.
     • If you are trying to use TypeScript, see https://jestjs.io/docs/getting-started#using-typescript
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/configuration
    For information about custom transformations, see:
    https://jestjs.io/docs/code-transformation

    Details:

    /home/prikhi/Projects/git/purescript-html-parser-halogen/test/main.test.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){import { main } from 'Test.Main';
                                                                                      ^^^^^^

    SyntaxError: Cannot use import statement outside a module

      at Runtime.createScriptFromCode (node_modules/jest-runtime/build/index.js:1796:14)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        0.564 s
Ran all test suites.```

Reply:
I see, let's merge this.

Reply:
Awesome, thank you :) `markdown-it-halogen` is next on my list :joy:

Reply:
Cool, I invited you as a collaborator. You can do a release if you want, thank you.